packages/wdio-allure-reporter/src/common/api.ts

```diff
@@ -20,8 +45,8 @@ const tellReporter = (event: string, msg: any = {}) => {
  * @param {string} name - label name
  * @param {string} value - label value
  */
-export function addLabel (name: string, value: string) {
-    tellReporter(events.addLabel, { name, value })
+export async function addLabel (name: string, value: string) {
```

Comment:
We can keep it fully backward compatible by omitting awaits, since the empty promise is used anyway